modules/core/src/main/java/org/apache/synapse/rest/RESTUtils.java

```diff
@@ -88,6 +92,48 @@ public static String getFullRequestPath(MessageContext synCtx) {
         return url;
     }
 
+    public static void mapQueryParamsToMessageProperties(MessageContext synCtx) {
```

Comment:
Shall we rename this as **populateQueryParamsToMessageContext** , ( that will describe more what this method intends to do , WDYT ?

Also shall we add a simple method level comment.